### PR TITLE
Bugfix for error when sdp is uninstalled

### PR DESCRIPTION
**Pull request by saranya-ag**
https://3.basecamp.com/3775257/buckets/16956142/todos/3606846770
https://3.basecamp.com/3775257/buckets/16956142/todos/3598578613

### CODE REVIEW

File: R/generate_template.R [hunk 1]
```diff
@@ -148,7 +148,7 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
             }
         }
         
-        if (!(.g_sdp_installed)) {
+        if (!(.g_sdp_installed) & dashboard_plus) {
```
Comment (agenius-mohammed-ali) [inline R/generate_template.R:151]: use normal && not bitwise &

**Review by agenius-mohammed-ali — CHANGES_REQUESTED** [review]
I suggest updating the condition to !.g_sdp_installed && dashboard_plus

File: R/generate_template.R [hunk 2]
```diff
@@ -148,7 +148,7 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
             }
         }
         
-        if (!(.g_sdp_installed)) {
+        if (!(.g_sdp_installed) && dashboard_plus) {
```
Comment (agenius-mohammed-ali) [inline R/generate_template.R:151]: no need for these extra brackets , just **!.g_sdp_installed && dashboard_plus** will be enough

Comment (cb4ds) [reply R/generate_template.R:151]: True - but I prefer more parenthesis than not, so I think this is fine.

**Review by cb4ds — APPROVED** [review]

**Pull request merged**